pkg/authorization/registry/clusterrole/proxy/proxy.go

```diff
@@ -81,11 +82,24 @@ func (s *ClusterRoleStorage) Create(ctx kapi.Context, obj runtime.Object) (runti
 	return authorizationapi.ToClusterRole(ret.(*authorizationapi.Role)), err
 }
 
-func (s *ClusterRoleStorage) Update(ctx kapi.Context, obj runtime.Object) (runtime.Object, bool, error) {
+type convertingObjectInfo struct {
```

Comment:
@liggitt I think this is the simplest way to solve this, but let me know if you disagree.